### ISSUE TITLE
atmel-rf-driver not coexisting without having .mbedignore -file

### ISSUE THREAD

**Issue by JanneKiiskila** [ARMmbed/atmel-rf-driver#59]
Same issue as with STM-Spirit1-RF - you can't have the atmel-rf-driver existing in the compilation path w/o an .mbedignore file without hitting compilation issues. This seems to be due to Nanostack missing, but if I'm not using any of these - the mbed_app.json in those cases should not need to have the NANOSTACK feature.

A compilation of mbed-os-example-client with K64F using the wifi-esp8266-v4.json will reproduce this.

```
gb-cam-ci-mbed-gw-build:~/mbed/mbed-os-example-client> mbed compile -m K64F -t GCC_ARM
Building project mbed-os-example-client (K64F, GCC_ARM)
Scan: .
Scan: env
Scan: mbed
Scan: FEATURE_STORAGE
Scan: FEATURE_COMMON_PAL
Scan: FEATURE_LWIP
Compile [  0.2%]: NanostackRfPhyAtmel.cpp
[Fatal Error] NanostackRfPhyAtmel.cpp@18,44: nanostack/platform/arm_hal_phy.h: No such file or directory
[ERROR] ./easy-connect/atmel-rf-driver/source/NanostackRfPhyAtmel.cpp:18:44: fatal error: nanostack/platform/arm_hal_phy.h: No such file or directory
 #include "nanostack/platform/arm_hal_phy.h"
                                            ^
compilation terminated.

[mbed] ERROR: "/usr/bin/python" returned error code 1.
[mbed] ERROR: Command "/usr/bin/python -u /home/jankii01/mbed/mbed-os-example-client/mbed-os/tools/make.py -t GCC_ARM -m K64F --source . --build ./BUILD/K64F/GCC_ARM" in "/home/jankii01/mbed/mbed-os-example-client"
---
```

**Comment by ciarmcom** [comment]
ARM Internal Ref: ONME-3342

**Comment by SeppoTakalo** [comment]
This will be fixed after IOTTHD-2035 is resolved.

**Comment by SeppoTakalo** [comment]
Fixed in #61
